Header header

geometry_msgs/Pose pose
geometry_msgs/Twist vel
geometry_msgs/Twist accel

int32 maxThrust

float64 GainCP

bool noInt
